installing/installing_azure/manually-creating-iam-azure.adoc

```diff
@@ -5,10 +5,26 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
```

Comment:
These changes have been reviewed in the other PR for 4.7 CCO work, but I copied them over since that one isn't merged yet.